var/spack/repos/builtin/packages/gimp/package.py

```diff
@@ -94,6 +96,10 @@ def url_for_version(self, version):
         url = "https://download.gimp.org/gimp/v{0}/gimp-{1}.tar.bz2"
         return url.format(version.up_to(2), version)
 
+    @when("@:2.10.32")
+    def patch(self):
+        filter_file("babl ", "babl-0.1 ", "configure")
```

Comment:
@wdconinc ditto - JFYI, Instead of this carrying this version constraint, I guess we could remove the older gimp versions (`gimp-2.10.32` and before) from the recipe if we have no need for them. (or make them deprecated?)

Reply:
Yes, let's mark as deprecated and remove after 0.23 is released.